google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/models/Filters.java

```diff
@@ -200,6 +200,7 @@ public Filter label(@Nonnull String label) {
   // Implementations of target specific filters.
   /** DSL for adding filters to a chain. */
   public static final class ChainFilter implements Filter {
+    private static final long serialVersionUID = -89237431180618430L;
```

Comment:
Here and everwhere else: The proto builders will need to be marked transient and you will need to implement writeObject/readObject to serialize the built protos:

```
void writeObject(ObjectOutputStream s) {
  s.defaultWriteObject();
  s.writeObject(builder.build());
}
```